eng/scripts/invoke_embedme.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Python version 3.4 or higher is required to run this script.
+
+# Use case: Invokes embedme README codesnippet generation and validation regardless of OS.
+#
+# Flags
+#   --readme/-r: Path to the README.
+#   --verify/-v: Flag indicating to only perform a dry-run validation.
+#   --debug/-d: Flag indicating to perform debug level logging.
+#
+# For example: Generating README codesnippets for Azure Storage Blobs.
+#    python eng/scripts/invoke_embedme.py -r sdk/storage/azure-storage-blob/README.md
+#
+# For example: Valdate README codesnippets for Azure Core.
+#    python eng/scripts/invoke_embedme.py -r sdk/core/azure-core/README.md -v
+#
+# The script must be run at the root of azure-sdk-for-java.
+
+import argparse
+import os
+import sys
+
+# NPX command for Windows OS.
+windows_command = 'npx.cmd'
+
+# NPX command for Posix OSes.
+posix_command = 'npx'
+
+# Invoke embedme.
+def invoke_embedme(readme: str, verify: bool, debug: bool):
+    command = ''
+    if os.name == 'nt':
+        command = windows_command
+    else:
+        command = posix_command
+
+    command += ' embedme'
```

Comment:
Don't you also need to install this command? 

I must say it is a little odd that we are using maven to call python to call npm lol :)

Reply:
Oh, I don't disagree on this oddity of Maven -> Python -> npm. Hopefully as our own tooling mature further these will be simplified into Maven plugins so that everything at least builds into Maven directly.